tools/packaging/kernel/configs/fragments/x86_64/sev/sev.conf

```diff
@@ -4,3 +4,9 @@ CONFIG_AMD_MEM_ENCRYPT=y
 CONFIG_AMD_MEM_ENCRYPT_ACTIVE_BY_DEFAULT=y
 CONFIG_CRYPTO_DEV_SP_PSP=y
 CONFIG_CRYPTO_DEV_CCP=y
+CONFIG_SECURITYFS=y
+CONFIG_VIRT_DRIVERS=y
+CONFIG_EFI=y
+CONFIG_EFI_SECRET=y
+CONFIG_MODULE_SIG=y
+CONFIG_MODULES=y
```

Comment:
@Alex-Carter01 please remind me why we set `CONFIG_EFI_SECRET=y` instead of `=m` ? Is it because modules are not copied into the initrd? If we're okay with it set to `=y` (built-in), then maybe there's no need to enable `CONFIG_MODULES` and `CONFIG_MODULE_SIG` ?

Reply:
I think this was a mistake on my end that `m` would be the correct option here. I am looking into this to make sure, but will probably switch to `m` soon. Good catch since I remember this coming up in a prior conversation but not sure where I regressed.